dependency_management.md

```diff
@@ -0,0 +1,20 @@
+Dependency Management
+===========
+
+The ResearchKit framework can also be added to your app using CocoaPods or Carthage.
+
+Installation with CocoaPods
+------------
+Add the following line to your [Podfile](http://guides.cocoapods.org/syntax/podfile.html) and run `pod install`:
+
+```ruby
+pod 'ResearchKit', '~> 1.0'
```

Comment:
Just to point out, this won't actually work until the Pod is pushed to the master spec repository.

One of the owners of this project should run `pod trunk push ResearchKit.podspec`.